cocos/core/math/mat4.ts

```diff
@@ -22,6 +22,8 @@
  THE SOFTWARE.
 */
 
+/* eslint-disable function-paren-newline */
```

Comment:
Why need it?

Reply:
Don't make eslint auto format to many lines for mat4.ts file.